website/docs/telemetry.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Telemetry
+---
+
+At Pact, we strive to develop great open source tools for the community. We rely on the community's input to help us iterate on and improve Pact. Telemetry is additional information that helps us to better understand the community's needs, diagnose issues, and prioritise feature work.
```

Comment:
Pact is not a place

Reply:
How can we word this better to refer to us as a collective? How about:

```suggestion
The Pact team strive to develop great open source tools for the community. We rely on the community's input to help us iterate on and improve Pact. Telemetry is additional information that helps us to better understand the community's needs, diagnose issues, and prioritise feature work.
```